game/scripts/npc/abilities/centaur_return.txt

```diff
@@ -26,7 +26,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "return_damage"                                   "16 32 48 64 80 100"
+        "return_damage"                                   "40 70 100 130 200 320" //OAA
```

Comment:
Leave Return Damage the same at Dota levels (Or a minor increase like 16/36/56/76, But I feel this is uneccesary). This hero will be impossible to trade with if you give him these values for return. Increase the damage on his Return Talent instead.

Reply:
It was very underwhelming in the test client. You shouldn't worry.